evergreen.yml

```diff
@@ -359,23 +347,24 @@ functions:
   # Expansions:
   #  cmake: The CMake executable to use
   #  toolchain: A toolchain directory to add to the PATH
-  ###
+  ##
   f_compile:
   - command: subprocess.exec
     params:
       working_dir: src
       binary: /bin/bash
       args:
-      - ${workdir}/src/scripts/bootstrap.sh
+      - "${workdir}/src/scripts/bootstrap.sh"
+      - "-DCMAKE_PREFIX_PATH=${workdir}/toolchain/shared/x64-${os_family}-shared"
+      - "-DCMAKE_TOOLCHAIN_FILE=${workdir}/toolchain/static/scripts/buildsystems/vcpkg.cmake"
+      - "-DVCPKG_TARGET_TRIPLET=x64-${os_family}-static"
       - "-DENABLE_JASPER:BOOL=ON"
-      - "-DGENNY_STATIC_BOOST_PATH:PATH=${workdir}/boost"
       - "-DGENNY_INSTALL_DIR:PATH=${workdir}/dist"
       - "-DGENNY_INSTALL_SSL:BOOL=ON"
       env:
         # This should be reasonably safe even on low memory machines
         MAKEFLAGS: "-j8"
         CMAKE: ${cmake|cmake}
-        PATH: ${toolchain}:/usr/bin:/bin
```

Comment:
...I forget why this PATH was necessary.

Reply:
Arh, because we need the compiler from the mongodb toolchain. I'm going to put it back in